SETUP.md

```diff
@@ -76,7 +76,7 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 ### Ubuntu 16.04 ([Download iso][ubuntu-iso-url]) Note: Virtual Machine Users should check the Windows Note below before starting
 
 1. `sudo apt-get update`
-1. `sudo apt-get install -y git mysql-server mysql-client libmysqlclient-dev libxslt1-dev libssl-dev zlib1g-dev imagemagick libmagickcore-dev libmagickwand-dev openjdk-9-jre-headless libcairo2-dev libjpeg8-dev libpango1.0-dev libgif-dev curl pdftk enscript libsqlite3-dev phantomjs build-essential redis-server rbenv npm`
+1. `sudo apt-get install -y git mysql-server mysql-client libmysqlclient-dev libxslt1-dev libssl-dev zlib1g-dev imagemagick libmagickcore-dev libmagickwand-dev openjdk-9-jre-headless libcairo2-dev libjpeg8-dev libpango1.0-dev libgif-dev curl pdftk enscript libsqlite3-dev phantomjs build-essential redis-server rbenv libreadline-dev node-js`
     * **Hit enter and select default options for any configuration popups, leaving mysql passwords blank**
 1. Install Node and Nodejs
     1. Type `curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -`
```

Comment:
Using the `apt` node-js probably contradicts this step installing nodejs from nodesource.com.

What really surprises me though is that our Ubuntu instructions aren't recommending just installing [`nvm`](https://github.com/creationix/nvm) which IMO is our best-supported approach to installing both node and npm.